### PR TITLE
[JENKINS-72711] Restore progress animation in build history and buildtime trend views

### PR DESCRIPTION
**Pull request by daniel-beck**
… time trend views

See [JENKINS-72711](https://issues.jenkins.io/browse/JENKINS-72711).

Amends #8705

### Testing done

Looked at the affected pages while builds were running while the previous build was: green, red, aborted, notbuilt, canceled.

Confirmed no other views include the responsible JS file (and would need these definitions).

### Proposed changelog entries

- Restore progress animation in build history and build time trend views (regression in 2.434).

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by mawinter69** [comment]
Not something for this PR but the many svg we include also blow up the html and usually most of them are not really needed (but one never knows). And I also have that in one of my plugins. So it would be nice to have the possibility to request a svg xml via fetch in javascript and just pass the symbol you want, e.g. `<jenkinsurl>/symbol?name=symbol-star-outline plugin-ionicons-api`
Or we provide directly a method and expose it to plugins.
getJenkinsSymbol("symbol-star-outline plugin-ionicons-api"), things like tooltip, htmltooltip, alt or id can be added as well

**Review by timja — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Comment by NotMyFault** [comment]
> Not something for this PR but the many svg we include also blow up the html and usually most of them are not really needed (but one never knows). And I also have that in one of my plugins. So it would be nice to have the possibility to request a svg xml via fetch in javascript and just pass the symbol you want, e.g. `<jenkinsurl>/symbol?name=symbol-star-outline plugin-ionicons-api` Or we provide directly a method and expose it to plugins. getJenkinsSymbol("symbol-star-outline plugin-ionicons-api"), things like tooltip, htmltooltip, alt or id can be added as well

Possibly worth filing a dedicated issue linking this PR?

---

/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
